DESCRIPTION.html

```diff
@@ -1,13 +1,16 @@
-Probably the sharpest git repository organizer and rebase/merge workflow automation tool you've ever seen!
+Probably the sharpest <b>git repository organizer</b> and <b>rebase/merge workflow automation</b> tool you've ever seen! <br/><br/>
```

Comment:
Will it work?

Reply:
Not sure... let's give it a shot. `<b>` works in `General Usage Instructions` and `<i>` works here in the description.